packages/boot/test/bootstrapTests/test-vats-restart.ts

```diff
@@ -141,31 +117,13 @@ test.serial('register network protocol before upgrade', async t => {
 });
 
 test.serial('run restart-vats proposal', async t => {
-  const { controller, buildProposal } = t.context;
+  const { controller, buildProposal, evalProposal } = t.context;
```

Comment:
Looks like `controller` is unused here.
```suggestion
  const { buildProposal, evalProposal } = t.context;
```

Reply:
I'll merge and try to remember to get the linter to find that next time